### PR TITLE
Add release not for vsts 2.*

### PR DESCRIPTION
**Pull request by Yiming-Jia**

### CODE REVIEW

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.1 - January, 21 2020
+  * Supported [Azure Active Directory authentication](https://github.com/Azure/AppConfiguration/blob/master/docs/REST/authorization/aad.md) of fetching key-values from App Configuration instance.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: This is a breaking change. I think we should provide more details to tell users how to migrate from what it was to what it is now (for example, before/after).

Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Updated, please take a look

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.9 - January, 21 2020
+  * **Breaking change**: supported [Azure Active Directory authentication](https://github.com/Azure/AppConfiguration/blob/master/docs/REST/authorization/aad.md) of fetching key-values from App Configuration instance.
+    * Change Service Connections'(Service Principals) role assignment to `Azure App Configuration Data Owner` or `Azure App Configuration Data Reader` role to grant pipeline Service Connection access to App Configuration access.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: How about this?

* **Breaking change**: With the support of Role-Based Access Control (RBAC) by the App Configuration service, the permission of this pipeline task can be granted with more granular and specific access control.

  **Before**
  The *Service Connection* of this pipeline task is granted `Contributor` role in access control of the App Configuration store.

  **After**
 The *Service Connection* of this pipeline task is granted `Azure App Configuration Data Reader` role in access control of the App Configuration store.

Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Updated

**Pull request merged**